src/data/mv_polynomial.lean

```diff
@@ -627,6 +638,14 @@ finset.sup_le $ assume b,
 
 end rename
 
+lemma eval₂_cast_comp {β : Type u} {γ : Type v} [decidable_eq β] [decidable_eq γ] (f : γ → β)
+  {α : Type w} [comm_ring α] (g : β → α) (x : mv_polynomial γ ℤ) :
+  eval₂ int.cast (g ∘ f) x = eval₂ int.cast g (rename f x) :=
```

Comment:
I guess we should we also remove these `int.cast`s?

Reply:
Okay, I think I got this last one now.